projects/react-boilerplate-ts/app/containers/App/index.tsx

```diff
@@ -21,7 +21,7 @@ export default function App() {
   return (
     <div>
       <Switch>
-        <Route exact path="/" component={HomePage} />
+        <Route exact={true} path="/" component={HomePage} />
```

Comment:
have we added some tslint rules that conflict with the default eslint rules that ship with boilerplate?
Personally I prefer the `<Route exact />` style of adding boolean values.